libs/vscode-ui/feature-task-execution-form/src/lib/task-execution-form.component.ts

```diff
@@ -331,8 +331,8 @@ export class TaskExecutionFormComponent implements OnInit, AfterViewChecked {
     dryRun?: boolean;
   }) {
     const flags = this.serializeArgs(form.value, architect);
+    flags.push('--no-interactive');
```

Comment:
Would it be better to use the defaults flag which disables interactive input prompts for options with a default. 

https://angular.io/cli/generate#options

Reply:
Hmm, yeah. Good question, nx should have this option too https://nx.dev/latest/angular/cli/generate#defaults